frontend/src/lib-customizations/employee.tsx

```diff
@@ -31,7 +34,7 @@ export {
   unitProviderTypes
 }
 
-export type Lang = 'fi'
+export type Lang = 'fi' | 'sv'
```

Comment:
I think it would make more sense to not add `'sv'` as a "UI language" since it doesn't seem to provide any benefit. What do you think?

Reply:
Combined with Kim's comments I think you might be right that it would be a good idea to not add Swedish here 👍